scripts/build_ckan_dataset_description.py

```diff
@@ -70,6 +70,9 @@ def build_ckan_dataset_description(strOutputFileName, number_of_objects, strFilt
     elif source == "worldpop":
         ckan_dataset_description["url"] = "https://hub.worldpop.org/"
         ckan_dataset_description["license_id"] = "cc-by"
+    elif source == "srtm":
+        ckan_dataset_description["url"] = "https://www.earthdata.nasa.gov/"
+        ckan_dataset_description["license_id"] = "Courtesy NASA/JPL-Caltech."
```

Comment:
I am afraid that it's not correct. Ckan has fixed number of licence ids. Also, "Courtesy" is not a license. What license is?

Reply:
done